src/sqlancer/postgres/gen/PostgresResetGenerator.java

```diff
@@ -0,0 +1,28 @@
+package sqlancer.postgres.gen;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import sqlancer.Randomly;
+import sqlancer.common.query.SQLQueryAdapter;
+import sqlancer.postgres.PostgresGlobalState;
+
+public final class PostgresResetGenerator {
+    private PostgresResetGenerator() {
+    }
+
+    public static SQLQueryAdapter create(PostgresGlobalState globalState) {
+        StringBuilder sb = new StringBuilder();
+        ArrayList<PostgresSetGenerator.ConfigurationOption> options = new ArrayList<>(
```

Comment:
I think we can simplify this logic using https://github.com/sqlancer/sqlancer/blob/30e10419d1ea85c6e03ab1517496cf3faa1e9468/src/sqlancer/Randomly.java#L116